### PR TITLE
Add prop that disables re-positioning on scroll

### PR DESCRIPTION
**Pull request by adelgado**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -380,7 +382,10 @@ const Popover = createReactClass({
     this.onPopoverResize = throttle(this.onPopoverResize, minResizeRefreshIntervalMs)
     this.onTargetResize = throttle(this.onTargetResize, minResizeRefreshIntervalMs)
 
-    this.frameEl.addEventListener("scroll", this.onFrameScroll)
+    if (this.props.repositionOnScroll) {
+      this.frameEl.addEventListener("scroll", this.onFrameScroll)
+    }
+
     resizeEvent.on(this.frameEl, this.onFrameResize)
```
Comment (Levino) [inline lib/index.js]: So will the popover resize / reposition if the frame size is changed?

Comment (adelgado) [inline lib/index.js]: Yes

Comment (Levino) [inline lib/index.js]: Is that what we want? I thought we want it to never be repositioned...

Comment (adelgado) [inline lib/index.js]: No, if the user resizes his window, we still want the popover to point at the correct target.

Comment (Levino) [inline lib/index.js]: Okay, I see.

**Pull request merged**